src/control/common/proto/ctl/network.pb.go

```diff
@@ -6,8 +6,8 @@
 
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.26.0
-// 	protoc        v3.6.1
+// 	protoc-gen-go v1.28.1
+// 	protoc        v3.5.0
```

Comment:
Please revert these changes in your tree and re-push. This is what I mentioned in the call the other day -- we don't commit these changes that are only for version updates because they cause a lot of review spam.

Reply:
Sorry, I didn't get this initially. I will change all the files from master which has only proto version changes.